tests/components/deconz/test_binary_sensor.py

```diff
@@ -80,6 +81,7 @@ async def test_binary_sensors(hass):
     assert "binary_sensor.clip_presence_sensor" not in gateway.deconz_ids
     assert "binary_sensor.vibration_sensor" in gateway.deconz_ids
     assert len(hass.states.async_all()) == 3
+    assert len(gateway.entities[binary_sensor.DOMAIN]) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)